From f996a84811c8d445c9d8612ee3f95d96a6fca8f9 Mon Sep 17 00:00:00 2001
From: sb6596 <shubhamprince111@gmail.com>
Date: Sat, 6 Oct 2018 00:38:10 -0400
Subject: [PATCH 2/2] sepolicy: Allow mm-qcamerad to access snap_app

* 10-05 04:31:31.318 471 471 W mm-qcamera-daem: type=1400 audit(0.0:22): avc: denied { use } for
  path="anon_inode:dmabuf" dev="anon_inodefs" ino=6711 scontext=u:r:mm-qcamerad:s0
  tcontext=u:r:snap_app:s0:c512,c768 tclass=fd permissive=0

This fixes panorama shots

Change-Id: I617d65d348b70174a8c0308332f50992f09ecd13
---
 legacy-common/mm-qcamerad.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/legacy-common/mm-qcamerad.te b/legacy-common/mm-qcamerad.te
index 977f071..abf82da 100644
--- a/legacy-common/mm-qcamerad.te
+++ b/legacy-common/mm-qcamerad.te
@@ -5,6 +5,7 @@ type_transition mm-qcamerad system_data_file:sock_file camera_socket "cam_socket
 allow mm-qcamerad camera_socket:sock_file { create unlink write };
 allow mm-qcamerad camera_data_file:sock_file { create unlink };
 allow mm-qcamerad platform_app:fd use;
+allow mm-qcamerad snap_app:fd use;
 allow mm-qcamerad system_server:unix_stream_socket rw_socket_perms;
 allow mm-qcamerad sensorservice_service:service_manager find;
 
-- 
2.17.1

